src/olympia/bandwagon/models.py

```diff
@@ -90,6 +90,8 @@ def publishable_by(self, user):
 class Collection(ModelBase):
     TYPE_CHOICES = amo.COLLECTION_CHOICES.items()
 
+    # TODO: Use models.UUIDField but it uses max_length=32 hex (no hypheh)
+    # uuids so needs some migration.
     uuid = models.CharField(max_length=36, blank=True, unique=True)
     name = TranslatedField(require_locale=False)
     # nickname is deprecated.  Use slug.
```

Comment:
yeah, but not in this PR :laughing: - it appears `nickname` is not much used but in some interesting places so maybe not too easy to remove or maybe it is, not sure.